driver-examples/src/main/java/com/datastax/driver/examples/opentelemetry/ZipkinUsage.java

```diff
@@ -41,8 +43,10 @@
  * <p>Side effects: - creates a new keyspace "simplex" in the cluster. If a keyspace with this name
  * already exists, it will be reused; - creates two tables "simplex.songs" and "simplex.playlists".
  * If they exist already, they will be reused; - inserts a row in each table.
+ *
+ * @param <ByteBuffer>
  */
-public class ZipkinUsage {
+public class ZipkinUsage<ByteBuffer> {
```

Comment:
This seems like it's been commited by mistake?

Reply:
You're right